### DISCUSSION TITLE
[Feature]: Support for Fujifilm RAF raw format

### DISCUSSION THREAD

**Discussion by AGJSpiegelenberg** [immich-app/immich#1600] [Feature Request]
### Feature detail

Currently RAF raw files are not being uploaded to the server. Will be great if this format can be supported.

### Platform

Server

**Comment by samip5** [comment]
Example file please so I can test what it thinks about it?

**Comment by AGJSpiegelenberg** [comment]
Fujifilm has two RAW formats, UNCOMPRESSED and LOSSLESS COMPRESSED. I added an example of both.
[Fujifilm - RAW Format - UNCOMPRESSED.zip](https://github.com/immich-app/immich/files/10487319/Fujifilm.-.RAW.Format.-.UNCOMPRESSED.zip)
[Fujifilm - RAW Format - LOSSLESS COMPRESSED.zip](https://github.com/immich-app/immich/files/10487325/Fujifilm.-.RAW.Format.-.LOSSLESS.COMPRESSED.zip)

**Comment by truongsinh** [comment]
From chat: https://discord.com/channels/979116623879368755/994044917355663450/1090733282892709988

> RAF support was already implemented in server / web
https://github.com/immich-app/immich/pull/1414

But CLI is not yet updated (tested today Mar 29)

**Reply by Thoroslives** [reply]
This is still an issue

I opened an issue on the cli git but no one has looked at it

**Comment by uhthomas** [comment]
Resolved by #2834.

**Reply by Thoroslives** [reply]
This is not solved, i just tried uploading some RAF files (not via cli as that still doesnt work) and they are just shown as broken images
![image](https://github.com/immich-app/immich/assets/78023877/fcd2fb52-2e7d-4e7f-a33c-66b8ed4b1199)

**Reply by uhthomas** [reply]
I see, thank you for letting us know. I'll create an issue.

```sh
immich_microservices     | [Nest] 2075  - 06/22/2023, 2:04:58 AM   ERROR [JobService] Unable to run job handler: Error: Input file contains unsupported image format
immich_microservices     | [Nest] 2075  - 06/22/2023, 2:04:58 AM   ERROR [JobService] Error: Input file contains unsupported image format
immich_microservices     | [Nest] 2075  - 06/22/2023, 2:04:58 AM   ERROR [JobService] Object:
immich_microservices     | {
immich_microservices     |   "id": "4ec8c8fb-764c-4201-a5d7-6b659a9585ce",
immich_microservices     |   "source": "upload"
immich_microservices     | }
```

**Reply by Thoroslives** [reply]
Thank you, you wouldnt know much about why the CLI doesnt support RAF?

**Reply by uhthomas** [reply]
The CLI is in need of some love. It doesn't support a lot of formats at the moment. #2824 has more info.

**Reply by AGJSpiegelenberg** [reply]
When using the Library feature this is not resolved.

**Reply by jaaacck** [reply]
Is this still on the roadmap to be added? I still see all my Fuji raw files showing as 'Error loading image' on the latest version
